onnx/checker.h

```diff
@@ -98,7 +98,8 @@ void check_node(
 void check_graph(
     const GraphProto& graph,
     const CheckerContext&,
-    const LexicalScopeContext&);
+    const LexicalScopeContext&,
+    bool is_main_graph=true);
```

Comment:
CheckerContext already has the "is_main_graph" attribute … it would be better to set/use that.

Reply:
Thanks for the comment, update to use CheckerContext.